src/Components/Form.js

```diff
@@ -0,0 +1,50 @@
+import { useState } from 'react';
+import { IconContext } from 'react-icons';
+import {AiFillPlusCircle}from 'react-icons/ai'
+import HandleUserChoice from './HandleUserChoice';
+const Form = ({addListToday,addListTomorrow,addListUpcoming}) => {
+    const [inputText,setInputText]=useState('');
+    const [showComponent, setShowComponent] = useState(false);
+    
+    return (
+        <div className="form">
+            <form>
+                
+                <input className="task"
+                        placeholder="I want to ..."
+                        value={inputText}
+                        type="text"
+                        onChange={e=>setInputText(e.target.value)}/>
+                
+                <IconContext.Provider  value={{color: '#00BFFF',size:'40'}}>
+                <div className='plusIcon'>
+                    <AiFillPlusCircle
+                         color='#00BFFF'
+                         size={40}
+                         onClick={() => {
```

Comment:
can you please extract these code lines into a new function and then invoke it here?